cmd/types.go

```diff
@@ -38,18 +38,20 @@ type Spec struct {
 	Token string `json:"token"`
 }
 
+// Provides basic user information required for
+// successful authentication.
+type User struct {
```

Comment:
Style: please put the declaration of the User struct after the Status struct just to follow original style.